devtools_m2/cypress/integration/RestApi.feature

```diff
@@ -0,0 +1,24 @@
+Feature: REST API Interactions
+
+  I want to make plugin settings available via the REST API
+
+  Scenario: Authorized request to set integration token
+    Given I am logged into the admin interface
+      And I have configured Drip to be enabled for 'main'
```

Comment:
I think most of these tests might be operating at a store level and giving us bad results. We might want to do a refactor PR to switch everything to work at a website level, although that could happen after this one (or before, with a rebase…)